Week3/normalization exercise/README.md

```diff
@@ -0,0 +1,12 @@
+# Exercise 1 : Normalization
+
+This exercise has questions like:
+
+- What columns violate 1NF?
+- Create a 3NF compliant solution.

```

Comment:
I don't understand this exercise. Do these questions refer to any problem ? any example table ?